nikola/utils.py

```diff
@@ -577,6 +581,14 @@ def __repr__(self):
                                                            sort_keys=True))
 
 
+def pkg_resources(package, resource):
```

Comment:
Please consider renaming this to something more descriptive and less similar to the thing it’s trying to replace, like `package_resource_path`.